### ISSUE TITLE
508-defect-2 [COGNITION, SCREEN READER]: Edit buttons SHOULD have accessible labels when editing inline data on Review screen

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17822]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The Edit buttons for contact information and issues eligible for review need an `aria-label` so screen reader users understand what they are editing without having to navigate back to the nearest heading. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Acceptance Criteria

- [ ] As a screen reader user, I want to hear an accessible label like "Edit issues eligible for review" instead of "Edit" when the button has focus or virtual cursor.

## Environment
* All screen readers
* All browsers

## Steps to Recreate

* Turn on a screen reader of your choice
* Enter `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/review-and-submit` in browser
* Follow test case 3
* Navigate to the **Review** page and open the accordions
* Click "Edit" under Issues Eligible for Review
* Verify you only hear "Edit" and have to navigate to the heading to answer "Edit what?"

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

<img width="935" alt="Screen Shot 2020-12-22 at 11 52 14 AM" src="https://user-images.githubusercontent.com/14154792/102913072-4160d380-444c-11eb-8aab-73d292abf18c.png">

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by luke-gcio** [comment]
Duplicate of https://github.com/department-of-veterans-affairs/va.gov-team/issues/17824
